frontend/src/metabase/core/components/TabLink/TabLink.tsx

```diff
@@ -7,7 +7,7 @@ import { TabContext, getTabId, getTabPanelId } from "../Tab";
 
 import { TabLabel, TabLinkRoot } from "./TabLink.styled";
 
-export interface TabLinkProps<T> extends LinkProps {
+export interface TabLinkProps<T> extends Omit<LinkProps, "value"> {
   value?: T;
 }
```

Comment:
Because of the change I made to `LinkProps`, we need to make this small change. No other changes are needed.